### DISCUSSION TITLE
UI Updates

### DISCUSSION THREAD

**Discussion by sass1997** [riok/Kreya#274] [Ideas]
I've started using Kreya these days. I'm using it only for rest api's. I was very confused to be forced to specify the base url into the endpoint input field which is hidden in settings. As a user which is mostly not reusing already configured operations this blocks me from using Kreya Ad Hoc. Additonally it would be great to have the possibility to configure the header key where the static value authorization is added to the requests. For example in gitlab you need to add the token into the header key "Private-Token"

**Comment by CommonGuy** [comment]
Hello and thanks for these ideas! You are right that Kreya does not yet support "REST only" use cases perfectly. In the upcoming 1.16 release, we improved many of these points.

You can now enter the full URL into the path field, which automatically extracts the endpoint into the settings. We feel that this is a good compromise, as having the endpoint in a separate field allows it to be changed much more easily.

Customizing the Auth header also comes with 1.16. You can try all of the improvements in the [beta release](https://kreya.app/docs/faq/#how-can-i-use-kreya-beta-versions)

**Comment by CommonGuy** [comment]
Sorry for the late answer. Your suggestions were implemented in the meantime. Try the latest version of Kreya to see it in action!
